main.py

```diff
@@ -1,7 +1,41 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
+genres = [
+    "Western",
+    "Action",
+    "Dramma",
+]
+
+actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson"),
+]
+
 
 def main() -> QuerySet:
-    pass
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for elem_first, elem_second in actors:
+        Actor.objects.create(
+            first_name=elem_first,
+            last_name=elem_second
+        )
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu", last_name="Reaves").update(
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue